editor/import/resource_importer_texture.cpp

```diff
@@ -181,15 +181,14 @@ bool ResourceImporterTexture::get_option_visibility(const String &p_option, cons
 }
 
 int ResourceImporterTexture::get_preset_count() const {
-	return 4;
+	return 3;
 }
 
 String ResourceImporterTexture::get_preset_name(int p_idx) const {
 	static const char *preset_names[] = {
-		"2D, Detect 3D",
+		"2D/3D (Auto-Detect)",
 		"2D",
-		"2D Pixel",
-		"3D"
+		"3D",
```

Comment:
Maybe remove the comma after "3D".

Reply:
I intentionally added it so we can get more VCS-friendly diffs in the future. clang-format currently doesn't enforce trailing commas in multiline arrays.